dotcom-rendering/src/web/components/MostViewedFooterGrid.tsx

```diff
@@ -169,35 +181,48 @@ export const MostViewedFooterGrid = ({
 									</span>
 
 									<TabHeading heading={tab.heading} />
-								</button>
+								</a>
 							</li>
 						);
 					})}
 				</ul>
 			)}
 			{data.map((tab: TrailTabType, i: number) => (
-				<ol
-					css={[gridContainer, i !== selectedTabIndex && hideList]}
+				<section
+					role={renderAsTabs ? 'tabpanel' : undefined}
 					id={`tabs-popular-${i}`}
-					data-cy={`tab-body-${i}`}
-					key={`tabs-popular-${i}`}
-					role="tabpanel"
-					aria-labelledby={`tabs-popular-${i}-tab`}
-					data-link-name={tab.heading}
-					data-testid={tab.heading}
-					data-link-context={`most-read/${sectionName}`}
+					key={`tabs-popular-${tab.heading}`}
 				>
-					{(tab.trails || []).map((trail: TrailType, ii: number) => (
-						<MostViewedFooterItem
-							key={trail.url}
-							position={ii + 1}
-							url={trail.url}
-							format={trail.format}
-							headlineText={trail.headline}
-							ageWarning={trail.ageWarning}
-						/>
-					))}
-				</ol>
+					{/*TODO: Decide whether it's h2 or h3*/}
+					<h3
+						css={css`
+							${visuallyHidden}
+						`}
+					>
+						Most viewed {tab.heading}
+					</h3>
+					<ol
+						css={[
+							gridContainer,
+							i !== selectedTabIndex && hideList,
+						]}
+						data-cy={`tab-body-${i}`}
+						data-link-name={tab.heading}
+						data-testid={tab.heading}
+						data-link-context={`most-read/${sectionName}`}
+					>
+						{tab.trails.map((trail: TrailType, ii: number) => (
+							<MostViewedFooterItem
+								key={trail.url}
+								position={ii + 1}
```

Comment:
I think generally we do `i` then `j`, rather than `ii`. I can see this was how it was done already, so no need to fix if you do not agree.